src/ansible_navigator/image_manager/puller.py

```diff
@@ -142,18 +151,33 @@ def prologue_stdout(self):
         print("Updating the execution environment")
         print("\u002d" * width)
 
+    def _generate_pull_command(self) -> List:
```

Comment:
Can we give the type constructor its parameter? Is it a `List[str]`?

Reply:
should be yes, nice.